x-pack/plugins/enterprise_search/public/applications/analytics/components/analytics_overview/analytics_collection_card/analytics_collection_card.styles.ts

```diff
@@ -0,0 +1,56 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { EuiThemeComputed } from '@elastic/eui/src/services/theme/types';
+
+export const AnalyticsCollectionCardStyles = (euiTheme: EuiThemeComputed) => ({
```

Comment:
This is a *lot* of CSS for us to be writing. We really try to minimize the amount of CSS we write because staying within the EUI component limitations makes everyone's lives a lot easier. Are there ways to minimize the amount of CSS we're using to do the styling here and elsewhere in this PR?

Reply:
I wish I could, but I'm afraid I can't much reduce it. At least if we want to follow design. In design we have some specific elements which can't be created only by Eui components. So I have to use mix of Eui components and own styles

Reply:
I can change some classes into
<img width="914" alt="image" src="https://user-images.githubusercontent.com/17390745/225461787-62af117f-b488-47b5-a8a7-ec754c9bb0e9.png">
What do you think?

Reply:
I talked to @joemcelroy at and as I understand it the long-term goal is going to be work with the Lens team to make their code more adaptable, which should solve this problem. For now this approach is probably fine. 

Using EUI mixins (and EUI-dependent variables generally) is slightly better, but the underlying issue is mostly that EUI components can change, and that can inadvertently break the CSS we're using without us noticing. 

I'll add that if design gives you things that are hard to implement with existing components, it's okay to push back where it makes sense. All of what we're implementing should be a conversation between design and engineering, and engineering constraints and concerns (including long-term maintainability) should be taken into account as well. Just because design hands you something doesn't mean that you need to exactly replicate that, if it's going to create significant problems or is unreasonably difficult. It's all about communicating.

Reply:
Agreed, it's if we rely on eui classes we dependence from their changes or just class renaming.
Julian is also coopreative and we had a meeting with him to change some design decisions

Reply:
Yeh its a happy compromise. I expect the designs to evolve more aligned with the design language of EUI data viz team in long term. We started the conversation with them and hopefully as both the project & data vis components matures, we will remove customisations :D